packages/rainbow-sprinkles/src/__tests__/createRainbowSprinkles.test.ts

```diff
@@ -173,6 +178,28 @@ describe('dynamic properties only', () => {
         }
       `);
     });
+
+    describe('handles all scale keys', () => {
```

Comment:
I've duplicated this block across the "dynamic properties only" and "static properties only" blocks as a sample, but it should probably be tested across all combinations (6 describe blocks). I didn't want to copy/paste this much test code. Do you have a preference here? Could abstract into helper function and run in each describe, or could add a new root level describe that does this specifically?

Reply:
I'm fine with limiting it to these two combinations. Thanks for adding the test!

```suggestion
    describe('parses scale values with hyphens, underscores, and periods', () => {
```